servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -5385,6 +5026,7 @@ public Response getSignedDomains(ResourceContext ctx, String domainName, String
         if (domainName != null) {
             domainName = domainName.toLowerCase();
             validate(domainName, TYPE_DOMAIN_NAME, caller);
+            setRequestDomain(ctx, domainName);
```

Comment:
Originally this method doesn't send the domain but I think it is correct to put it (if it exists)